bin/install

```diff
@@ -2,6 +2,13 @@
 
 set -euo pipefail
 
+semver_lte() {
```

Comment:
This function is not necessary and can be simpler by stripping out the dots and comparing them as integers.